contracts/aggchains/AggchainECDSAMultisig.sol

```diff
@@ -10,7 +11,7 @@ import "../lib/AggchainBase.sol";
  * transitions on the pessimistic trees (local_exit_tree, local_balance_tree, nullifier_tree & height).
  * The addresses and threshold are managed by the aggchainManager.
  */
-contract AggchainECDSAMultisig is AggchainBase {
+contract AggchainECDSAMultisig is AggchainBase, MigrationFEPToECDSASlots {
```

Comment:
do you both @invocamanman and @krlosMata agree on this approach?

Reply:
I think it is not needed, although is does not hurt.
A comment should be enough since the storage slots fulfilled are just not used.
It is needed a deeper analysis if the chain goes again to FEP. Check if the storage slots are rewritten or can potentially imply a security issue